cmd/scollector/collectors/processes_windows.go

```diff
@@ -10,31 +10,28 @@ import (
 	"bosun.org/opentsdb"
 )
 
-var regexes = []*regexp.Regexp{}
+var regexesProcesses = []*regexp.Regexp{}
 
 func AddProcessConfig(line string) error {
 	reg, err := regexp.Compile(line)
```

Comment:
This was existing code, but I think it should Panic if something in the config isn't a valid regex. You can use `regexp.MustCompile` to do this